packages/web-client/app/components/card-pay/deposit-workflow/index.ts

```diff
@@ -10,12 +10,14 @@ import Layer1Network from '@cardstack/web-client/services/layer1-network';
 import Layer2Network from '@cardstack/web-client/services/layer2-network';
 import { inject as service } from '@ember/service';
 import { action } from '@ember/object';
+import { currentNetworkDisplayInfo as c } from '@cardstack/web-client/utils/web3-strategies/network-display-info';
+import { capitalize } from '@ember/string';
 
 class DepositWorkflow extends Workflow {
   name = 'Reserve Pool Deposit';
   milestones = [
     new Milestone({
-      title: 'Connect mainnet wallet',
+      title: `Connect ${c.layer1.networkType} wallet`,
```

Comment:
Unsure about replacing mainnet with testnet.

Reply:
Yeah, it is not clear, because both sokol and kovan are testnets.

Reply:
Will check with design about whether we can replace 'mainnet' with the full testnet name.

Reply:
Kovan's name will read `Kovan testnet`.